tests/test_cli.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
+"""Test mdacli cli."""
+import subprocess
+import sys
+from unittest.mock import patch
+
+import pytest
+from MDAnalysis.analysis.rdf import InterRDF
+from MDAnalysisTests.datafiles import DCD, PSF
+
+from mdacli.cli import run_analsis, setup_clients
+
+
+def test_required_args():
+    """Test that there is a module given."""
+    with pytest.raises(subprocess.CalledProcessError):
+        subprocess.check_call(['mdacli'])
+
+
+def test_wrong_module():
+    """Test for a non existent module."""
+    with pytest.raises(subprocess.CalledProcessError):
+        subprocess.check_call(['mdacli', 'foo'])
+
+
+@pytest.mark.parametrize('args', ("version", "help"))
```

Comment:
Interesting... I never thought of testing the actual call to the command line. That is good. Keep in mind this works because of the way `tox` installs the package. Which is good :wink:

Reply:
Jippie :)